cli/fmt.rs

```diff
@@ -29,10 +27,11 @@ fn is_supported(path: &Path) -> bool {
     && (TYPESCRIPT.is_match(&path_str) || JAVASCRIPT.is_match(&path_str))
 }
 
-fn get_config() -> Configuration {
-  ConfigurationBuilder::new()
+fn get_config() -> dprint::Configuration {
+  dprint::ConfigurationBuilder::new()
     .line_width(80)
     .indent_width(2)
+    .next_control_flow_position(dprint::NextControlFlowPosition::SameLine)
```

Comment:
You will also want:

```rust
.force_multi_line_parameters(true)
.force_multi_line_arguments(true)
```

See https://dprint.dev/#forcemultilinearguments (Perhaps these config options are poorly named)

Reply:
Opened https://github.com/dsherret/dprint/issues/63